cpp/src/io/csv/csv_reader.cu

```diff
@@ -888,20 +892,21 @@ __global__ void countRecords(char *data, const char terminator, const char quote
 gdf_error launch_storeRecordStart(raw_csv_t * csvData) {
 
 	int blockSize;		// suggested thread count to use
-	int minGridSize;	// minimum block count required
-	CUDA_TRY( cudaOccupancyMaxPotentialBlockSize(&minGridSize, &blockSize, storeRecordStart) );
+ 	int minGridSize;	// minimum block count required
+ 	CUDA_TRY( cudaOccupancyMaxPotentialBlockSize(&minGridSize, &blockSize, storeRecordStart) );
 
 	// Calculate actual block count to use based on bitmap count
-	// Each bitmap is for a 64-byte chunk, and each data index is bitmap ID * 64
-	int gridSize = (csvData->num_bits + blockSize - 1) / blockSize;
+ 	// Each bitmap is for a 64-byte chunk, and each data index is bitmap ID * 64
+ 	int gridSize = (csvData->num_bits + blockSize - 1) / blockSize;
 
 	storeRecordStart <<< gridSize, blockSize >>> (
-		csvData->data, csvData->terminator, csvData->quotechar,
-		csvData->num_bytes, csvData->num_bits, csvData->d_num_records,
-		csvData->recStart
-	);
+ 		csvData->data, csvData->terminator, csvData->quotechar,
+ 		csvData->num_bytes, csvData->num_bits, csvData->d_num_records,
+ 		csvData->recStart
+ 	);
+
+ 	CUDA_TRY( cudaGetLastError());
```

Comment:
Since the previous PR was accepted, merge conflicts resulted in having to manually copy the previous changes back, hence the unintentional extra spaces. Will proceed to fix these trivial things.